partcad-cli/src/partcad_cli/click/commands/info.py

```
@@ -0,0 +1,65 @@
+@click.option(
+    "-p",
+    "--param",
+    "params",
+    type=str,
+    multiple=True,
+    metavar="<name>=<value>",
+    help="Assign a value to the parameter",
+)
+@click.argument("object", type=str, required=False)  # help="Part (default), assembly or scene to show"
+@click.pass_obj
+def cli(ctx, package, interface, assembly, sketch, scene, object, params):  # , path
```

Ruff (0.8.2)

`cli` is too complex (11 > 10)

(C901)